sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -480,6 +486,76 @@ def test_with_input_types_decorator_violation(self):
         pc = p | Create(l_3_tuple)
         _ = pc | beam.CombineGlobally(self.fn)
 
+#
+# Test cases for streaming.
+#
+@attr('ValidatesRunner')
+class TimestampCombinerTest(unittest.TestCase):
+
+  class RecordFn(beam.DoFn):
+    def process(self, elm=beam.DoFn.ElementParam, ts=beam.DoFn.TimestampParam):
+      yield (elm, ts)
+
+  @unittest.skip('BEAM-8657')
+  def test_combiner_earliest(self):
+    """Test TimestampCombiner with EARLIEST."""
+    options = PipelineOptions(streaming=True)
+    with TestPipeline(options=options) as p:
+      result = (p
+                | 'main TestStream' >> TestStream()
```

Comment:
Omit the `label >>` unless they are needed for uniqueness or clarity. These labels don't add anything to what the default labels provide.

Reply:
Thanks!  Will double check the style on all the other pending PRs as well.